yaml/xyz/openbmc_project/Inventory/Item/PCIeSlot.interface.yaml

```diff
@@ -25,6 +25,12 @@ properties:
       description: >
           Whether this PCIe slot supports hotplug
 
+    - name: LinkStatus
+      type: enum[self.Status]
+      default: 'unknown'
```

Comment:
Should this be 'Unknown'  matches with one of the values for Status ?

Reply:
@SunnySrivastava1984 build is failing due to incorrect default value for LinkStatus...

Reply:
> @SunnySrivastava1984 build is failing due to incorrect default value for LinkStatus...

Ahh, yes. I missed to update that. Thanks.

Reply:
> > @SunnySrivastava1984 build is failing due to incorrect default value for LinkStatus...
> 
> Ahh, yes. I missed to update that. Thanks.

We need to get this PR merged - I've dependency on this; There seems to be a merge conflict...